### DISCUSSION TITLE
AutoUpdate enable?

### DISCUSSION THREAD

**Discussion by gregoryvanko** [coder/code-server#2240] [Q&A] [answered]
@code-asher, My AutoUpdate seems to not working after release 3.5.0, is it a parameter to set?

For the moment I execute each time 2 commands to update code-server to the new version : 
curl -fsSL https://code-server.dev/install.sh | sh)
sudo systemctl enable --now code-server@$USER

do i miss something to have update automatically download and install?
Thank you.

**Comment by code-asher** [comment, marked as answer]
We removed auto-updates so the official way to update is to run the
install script again like you're doing here. You shouldn't have to
enable the service again but you will need to restart it.

**Reply by yiliang114** [reply]
The main reason for deleting the automatic update function is that it may affect users' data and files, and to some extent, may cause the update to be unsuccessful due to permission problems?

**Reply by code-asher** [reply]
I think that was part of it, yeah.
